app/src/main/java/org/sopt/kream/presentation/ui/main/home/recommend/RecommendCircleMenuAdapter.kt

```diff
@@ -0,0 +1,33 @@
+package org.sopt.kream.presentation.ui.main.home.recommend
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import org.sopt.kream.databinding.ItemRecommendCircleMenuBinding
+
+class RecommendCircleMenuAdapter(val list: List<String>) : RecyclerView.Adapter<RecommendCircleMenuAdapter.ViewHolder>() {
+    val item = list
```

Comment:
```suggestion
    private val item = list
```

외부에서 접근하지 않는 변수는 무조건 private 로 선언해 줍시다.
추가적으로 backing properties에 대해서 공부해 보시면 좋을 것 같아요